glean_parser/templates/csharp.jinja2

```diff
@@ -0,0 +1,97 @@
+// -*- mode: csharp -*-
+
+/*
+ * AUTOGENERATED BY glean_parser.  DO NOT EDIT.
+ */
+{# The rendered markdown is autogenerated, but this
+Jinja2 template is not. Please file bugs! #}
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+{% macro obj_declaration(obj, suffix='', access='', lazy=False) %}
+{{ access }} {% if lazy %} Lazy<{{ obj|type_name }}>{%- else %} {{ obj|type_name }}{% endif %} {{ obj.name|camelize }}{{ suffix }}{% if lazy %} = new Lazy<{{ obj|type_name }}>(() => {%- else %} ={% endif %}
```

Comment:
nit (only if not a quagmire): You can use `{%-` and `-%}` around commands to remove whitespace which means you don't have to have a big run-on line like this.